custom_components/ha_kia_hyundai/select.py

```diff
@@ -4,19 +4,18 @@
 from dataclasses import dataclass
 from typing import Final
 
-from custom_components.ha_kia_hyundai import VehicleCoordinator
-from custom_components.ha_kia_hyundai.vehicle_coordinator_base_entity import (
-    VehicleCoordinatorBaseEntity,
-)
```

Comment:
Happened to notice these could be imported using the . shorthand so they are moved below and everything sorted